app/master/cluster_master.py

```diff
@@ -329,7 +326,7 @@ def get_path_for_build_results_archive(self, build_id: int, is_tar_request: bool
         :param is_tar_request: If true, download the tar.gz archive instead of a zip.
         :return: The path to the archived results file
         """
-        build = self._all_builds_by_id.get(build_id)  # type: Build
+        build = BuildStore.get(build_id)  # type: Build
```

Comment:
I believe you can remove the typehint comment here now since `BuildStore.get` has its own hint.